tools/cdf-lineage-exporter/tests/unit/test_client.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Google LLC
```

Comment:
remove empty test file or add unit test here.

Reply:
Will do. There's a reason I opened this in draft mode. 😉

Reply:
yeah sorry about the premature review!